Changelog.md

```diff
@@ -6,6 +6,7 @@ Language Features:
 
 Compiler Features:
  * Commandline Interface: Use proper severity and coloring also for error messages produced outside of the compilation pipeline.
+ * EVM: Deprecate support for "Constantinople" and older EVM versions.
```

Comment:
Should I mention the deprecation in the docs?

Reply:
https://docs.soliditylang.org/en/develop/using-the-compiler.html#target-options would seem like an optimal place, wouldn't it?

Reply:
Done.